flickr_download/filename_handlers.py

```diff
@@ -81,3 +85,13 @@ def get_filename_handler(name='title'):
     @return: Function, handler
     """
     return HANDLERS[name]
+
+
+def get_foldername(photo, pattern):
+    """
+    Returns date based foldername
```

Comment:
needs docstring for arguments and return value. Also this file is for functions that determine the file names, put it in the mail `.py` file